src/components/activity-page/activity-page-content.scss

```diff
@@ -32,6 +32,13 @@
     }
 
     .group {
+      &.responsive {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: space-between;
+      }
+
```

Comment:
This is the main piece that handles the side-by-side layout of the non-full-width questions.